### PR TITLE
[Feature][e2e][Clickhouse] Add Clickhouse e2e testcase

### PR DESCRIPTION
**Pull request by FWLamb**
## Purpose of this pull request
Add Clickhouse e2e testcase https://github.com/apache/incubator-seatunnel/issues/2949

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Review by Hisoka-X — DISMISSED** [review]
LGTM

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+        clearSinkTable();
+    }
+
+    @BeforeAll
+    @Override
+    public void startUp() throws Exception {
+        this.container = new ClickHouseContainer(CLICKHOUSE_DOCKER_IMAGE)
+            .withNetwork(NETWORK)
+            .withNetworkAliases(HOST)
+            .withLogConsumer(new Slf4jLogConsumer(LOG));
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]:
```suggestion
            .withLogConsumer(new Slf4jLogConsumer(DockerLoggerFactory.getLogger(CLICKHOUSE_DOCKER_IMAGE)));
```

https://github.com/apache/incubator-seatunnel/pull/3028

**Review by hailin0 — COMMENTED** [review]
LGTM

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java (hunk 1 above)
Comment (FWLamb) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java]: copy, I do it.

**Review by ashulin — APPROVED** [review]
LGTM

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**